packages/create-cloudflare/scripts/e2eCleanup.ts

```diff
@@ -14,6 +20,7 @@ const run = async () => {
 	const projectsToDelete = (await listC3Projects()) as Project[];
 
 	for (const project of projectsToDelete) {
+		console.log("Deleting Pages project: " + project.name);
 		await deleteProject(project.name);
 	}
```

Comment:
This will conflict across parallel e2e test runs? I think it'd be more accurate to keep track of projects created in this specific run and delete just those

Reply:
yeah we did discuss this in our team meeting today and are going to improve the solution so that it is PR specific as you suggested 🙂:+1: 
(but for this PR I would think that it's ok to just keep it as is, consistent with how we also clean the pages projects up and fix both together later on)